src/pages/Home.jsx

```diff
@@ -2,7 +2,7 @@
 import { Container } from '@mui/material';
 // Component Imports
 import Hero from '../components/global/Hero/Hero';
-import About from '../components/home/About';
+import About from '../components/home/about/About';
```

Comment:
Since the `About` section is now split up into separate components, we could route them all into an `index.js` file for best practice.

Definitely not necessary but wanted to mention. We're not doing it elsewhere, anyway.

Reply:
I don't see this as necessary, these aren't very complex and its a small project.